configure.ac

```diff
@@ -1510,10 +1510,10 @@ AC_CONFIG_FILES([Makefile
                  tests/API/XCCDF/variable_instance/Makefile
 
                  tests/schemas/Makefile
-		tests/bz2/Makefile
-		tests/codestyle/Makefile
-		tests/oval_details/Makefile
-		tests/nist/Makefile
+                 tests/bz2/Makefile
+                 tests/codestyle/Makefile
+                 tests/oval_details/Makefile
+                 tests/nist/Makefile
 
                  src/SCE/Makefile
                  tests/sce/Makefile])
```

Comment:
Please don't change indents, this just creates merge conflicts for no real gain. Plus we use tabs, not spaces.